caffe2/utils/math_cpu.cc

```diff
@@ -769,6 +769,15 @@ DELEGATE_CUBE_FUNCTION(std::int32_t)
 DELEGATE_CUBE_FUNCTION(std::int64_t)
 #undef DELEGATE_CUBE_FUNCTION
 
+#define DELEGATE_INV_FUNCTION(T)                                        \
+  template <>                                                           \
+  void Inv<T, CPUContext>(const int N, const T* x, T* y, CPUContext*) { \
```

Comment:
Consider to use vsInv and vdInv in mkl when CAFFE2_USE_MKL defined.
https://software.intel.com/en-us/mkl-developer-reference-c-v-inv

Reply:
ok